### PR TITLE
LRCI-978 Add methods to create spira releases

### PR DESCRIPTION
**Pull request by michaelhashimoto**
https://issues.liferay.com/browse/LRCI-978

### CODE REVIEW

**Comment by michaelhashimoto** [comment]
ci:test:sf

**Comment by michaelhashimoto** [comment]
ci:test:relevant

File: modules/test/jenkins-results-parser/src/main/java/com/liferay/jenkins/results/parser/spira/SpiraRelease.java [hunk 1]
```diff
@@ -33,6 +36,119 @@
...
+		String releaseName = releasePathNames[releasePathNames.length - 1];
+
+		String parentReleasePath = sb.toString();
+
+		if (parentReleasePath.isEmpty()) {
+			return createSpiraRelease(spiraProject, releaseName);
+		}
+
+		SpiraRelease parentSpiraRelease = createSpiraReleaseByPath(
+			spiraProject, parentReleasePath);
```
Comment (pyoo47) [inline modules/test/jenkins-results-parser/src/main/java/com/liferay/jenkins/results/parser/spira/SpiraRelease.java]: Shouldn't the `parentReleasePath` drop the last item in `releasePathNames[]`, not the first?
```
   		for (int i = 0; i < (releasePathNames.length - 2); i++) {
			sb.append("/");
			sb.append(releasePathNames[i]);
		}
```

**Comment by pyoo47** [comment]
Pushed to BChan
https://github.com/brianchandotcom/liferay-portal/pull/84298

@michaelhashimoto Please see my changes in the pull request.

**Pull request closed without merging**